tests/optional_chaining/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`chaining.js 1`] = `
+var street = user.address?.street
+var fooValue = myForm.querySelector('input[name=foo]')?.value
+
+obj?.prop;
+obj?.[expr];
+func?.(...args);
+
+a?.();
```

Comment:
That's the syntax [currently in the proposal](https://github.com/tc39/proposal-optional-chaining#syntax), but there is no "actual syntax" here - this is a stage 1 proposal, which amounts to "someone on the committee thought something kinda vaguely like this might be a good idea". Its syntax and semantics are likely to change radically and repeatedly before it makes it into the language, if it ever does.

That said, `a?()` in particular is unlikely because it's ambiguous: `a?(0)` could be a call, or could be the first two thirds of a ternary.